drivers/interrupt_controller/Kconfig.uwp

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright (c) 2018, UNISOC Incorporated
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig UWP_ICTL
+	bool "UNISOC Interrupt Controller Driver"
+	depends on SOC_FAMILY_UWP
+	depends on MULTI_LEVEL_INTERRUPTS
+	help
+	  Enable UNISOC uwp5 interrupt controller.
+
+if UWP_ICTL
+
+config UWP_ICTL_0
```

Comment:
seems like these shouldn't be numbers, but named by irq purpose.

Reply:
Sorry I don't get it. These are individual instances so should be named as `0, 1`. I'll rename `ICTL_3` to `ICTL_AON`